scripts/dts/extract/globals.py

```diff
@@ -351,38 +351,63 @@ def get_binding(node_address):
 def get_binding_compats():
     return bindings_compat
 
+def build_cell_array(prop_array):
+
+    index = 0
+    ret_array = []
+
+    while index < len(prop_array):
+        handle = prop_array[index]
+
+        if handle in {0, -1}:
+            ret_array.append([])
+            index += 1
+        else:
+            # get controller node (referenced via phandle)
+            cell_parent = phandles[handle]
+
+            for k in reduced[cell_parent]['props'].keys():
+                if k[0] == '#' and '-cells' in k:
+                    num_cells = reduced[cell_parent]['props'].get(k)
```

Comment:
The rest of the code assumes `k` exists, so could do `[k]` instead of `.get(k)`.

Could add a `break` too maybe. The rest of the code will crash if there's no `#...-cells`  property either way though.

Reply:
Done